src/js/components/modals/CliInstallModal.js

```diff
@@ -77,11 +77,10 @@ class CliInstallModal extends React.Component {
 
     const instructions = [
       `curl ${downloadUrl} -o dcos`,
-      "sudo mv dcos /usr/local/bin",
-      "sudo chmod +x /usr/local/bin/dcos",
-      `dcos config set core.dcos_url ${clusterUrl}`,
-      Hooks.applyFilter("dcosInstallCommandExtraSteps", undefined),
-      "dcos"
+      `sudo mv dcos /usr/local/bin`,
+      `sudo chmod +x /usr/local/bin/dcos`,
+      `dcos cluster setup ${clusterUrl}`,
+      `dco`
```

Comment:
good catch. misspelled addressed.

Now the extra steps part looks like we don't need anymore based on the Jira ticket https://jira.mesosphere.com/browse/DCOS-14334?